src/util/registry_utils.js

```diff
@@ -91,3 +99,29 @@ export function getEntry(name, branch = "master") {
   }
   return null;
 }
+
+async function getGithubLatestVersion(owner, repo) {
+  const response = await fetch(
+    `https://api.github.com/repos/${owner}/${repo}/releases/latest`
+  );
```

Comment:
We can't do this from the worker as we would very quickly run into Github API rate limits. Limits are 60 req per hour for non authenticated users, and 3000 for authenticated users. Because we share the ip range where the worker is requesting from with other people (because that's how CF workers work), we will be through this limit permanently. IDK what the current statistics for deno.land/x are, but definitely more than 60 req per hour.

The only reason we can call the API for the website without running into limits is because we do this from the client.

I don't see any other way of doing this though, so it might be worth investigating if Github is willing to raise our rate limit to enable this - idk if there is precedent for that though.

Reply:
isn't the hack to parse the releases rss feed: e.g. https://github.com/denoland/deno/releases.atom

Reply:
Yup. We might want to consult with Github first to see what they think is the right way to go here - I presume this API request is cheaper for them than requesting this large XML file. It might be in their own best interest to give us an increased rate limit here.

Reply:
> isn't the hack to parse the releases rss feed: e.g. https://github.com/denoland/deno/releases.atom

I can't fetch that page from another site due to the CORS policy, try `await (await fetch("https://github.com/denoland/deno/releases.atom")).text()`. Know a good workaround?

Reply:
Nope - can't really easily work around CORS. You can proxy via the worker maybe?